tensorflow/python/saved_model/loader_test.py

```diff
@@ -290,6 +290,13 @@ def test_load_saved_model_graph_with_return_elements(self, builder_cls):
     with self.assertRaisesRegex(ValueError, "not found in graph"):
       loader.load_graph(graph, ["foo_graph"], return_elements=["z:0"])
 
+  def test_parse_saved_model_exception(self, builder_cls):
+    """Test that error message for not exist model have OS-depend delimiter in path"""
+    path = _get_export_dir("not_existing_dir")
+    pattern = os.path.sep + "{"
+    with self.assertRaises(IOError) as err:
+      loader_impl.parse_saved_model(path)
+    self.assertTrue(pattern in str(err.exception))
```

Comment:
Is this not going to match `os.path.sep` in `_get_export_dir` and so in `test.get_temp_dir()`?

Reply:
No, in this test we search correct delimiter between directory name and "`{`".
Error message formed by pattern "`SavedModel file does not exist at: %s%s{%s|%s}" (in current version "SavedModel file does not exist at: %s/{%s|%s}`").
For example, "`SavedModel file does not exist at: C:\IncorrectPathToSavedModel\{saved_model.pbtxt|saved_model.pb}`".
We checking combination from delimiter and "`{`", path to temporary dir can't have this combination, it's simple test for correct path separator detection.
IMHO, this change can to be accepted without new tests, but I respect rules of project and created test for regression. For current "main" branch this test is red, for updated code - green, I think it's correct.
Excuse me if I incorrect understand your comment and give not valid answer, I want to make my code better, if I can.

Reply:
Ok `os.path.sep + "{"`